engine/rpcserver.go

```diff
@@ -2201,12 +2193,7 @@ func (s *RPCServer) GetTickerStream(r *gctrpc.GetTickerStreamRequest, stream gct
 			return errDispatchSystem
 		}
 
-		d := *data.(*interface{})
-		if d == nil {
-			return errors.New("unable to type assert data")
-		}
-
-		t, ok := d.(ticker.Price)
+		t, ok := data.(ticker.Price)
```

Comment:
go run ./... gettickerstream -exchange="btc markets" -pair="USDC-AUD" -asset='spot'
2022/04/14 12:27:50 rpc error: code = Unknown desc = unable to type assert ticker data